### PR TITLE
Docs release notes 10 24 24

### PR DESCRIPTION
**Pull request by paperclypse**

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+subject: "Docs"
+releaseDate: '2024-10-25'
+version: 'version: October 18-24, 2024'
+---
+
+### New docs
+
+* New troubleshooting doc [PHP agent overhead reduction tips](/docs/apm/agents/php-agent/troubleshooting/php-agent-overhead-reduction-tips) lists configuration settings you can change to reduce the agent's overhead.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx:9]: Suggestion: We can drop "New troubleshooting doc "  to maintain consistency with other sections.

[PHP agent overhead reduction tips](/docs/apm/agents/php-agent/troubleshooting/php-agent-overhead-reduction-tips) lists configuration settings you can change to reduce the agent's overhead.

The link seems to be broken

File: src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+### Major changes
+
+* We've released our new Advanced Compute add-on and updated various pricing, billing, user, and product license docs to reflect this.
+* We've revamped our Docs site search to give you a separate search results page. Check it out and let us know what you think!
+* Updated our Kubernetes agent operator doc to include information on [Kubernetes APM auto-attach](/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator).
+
+### Minor changes
+
+* When [testing the Python agent](/docs/apm/agents/python-agent/troubleshooting/testing-python-agent/#host-name-lookup) to troubleshoot collector issues, updated outdated hostname lookup IP addresses.
+* In [instrument your non-containerized Lambda function](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/instrument-your-own/#cdk), added example code for a basic CDK app that deploys a Node.js Lambda function instrumented by New Relic.
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx:20]: Suggestion: In [instrument your non-containerized Lambda function](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/instrument-your-own/#cdk), added example code for a basic CDK app that deploys a Node.js.

We can remove redundant trailing words like "Lambda function instrumented by New Relic".

Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx:20]: Absolutely right!

Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx:20]: I do think we need to mention the Lambda function there, even if it's a bit redundant with the doc title

File: src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    * Updates native iOS agent to version 7.5.2
+* [Flutter agent v1.1.5](/docs/release-notes/mobile-release-notes/flutter-release-notes/flutter-agent-115/)
+    * Updates native Android agent to version 7.6.1
+* [.NET MAUI agent v1.1.3](/docs/release-notes/mobile-release-notes/net-maui-release-notes/net-maui-agent-113/)
+    * Fixes iOS binding information property list issue
+* [.NET MAUI agent v1.1.4](/docs/release-notes/mobile-release-notes/net-maui-release-notes/net-maui-agent-114/)
+    * Updates native Android agent to version 7.6.1
+    * Updates native iOS agent to version 7.5.2
+    * Adds support for handling iOS crash exceptions
+* [Node.js agent v12.5.2](/docs/release-notes/agent-release-notes/nodejs-release-notes/nodejs-agent-12-5-2/)
```
Comment (cbehera-newrelic) [inline src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx:53]: The link seems to be broken

**Review by cbehera-newrelic — COMMENTED** [review]
Left minor comments and suggestion. Rest looks good to me

File: src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx:53]: Whoops!

**Review by cbehera-newrelic — APPROVED** [review]
LGTM

**Pull request merged**